.github/workflows/fetch-icons.yaml

```
@@ -0,0 +1,112 @@
+###
+#
+# Create a pull request with up-to-date icon SVGs
+#
+# This workflow does not have secrets access
+# when run from a fork. Artifacts are uploaded
+# and used in a subsequent workflow with more
+# privileges which does not run unsafe commands.
+#
+# https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
+#
+###
+
+name: Fetch Icons
+
+on:
+  schedule:
+    - cron: '* * * * 0'
+
+jobs:
+  fetch:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup
+        uses: swisspost/design-system/.github/actions/setup-pnpm@main
+
+      - name: Get Date
+        id: current-date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+
+      - name: Create Branch
+        id: current-branch
+        run: |
+          git checkout -b $BRANCH_NAME
+          echo "::set-output name=branch::$BRANCH_NAME"
+        env:
+          BRANCH_NAME: ${{ steps.current-date.outputs.date }}-update-icon-svgs
+
+      - name: Fetch Icons
+        run: pnpm --filter design-system-icons fetchSVGs
+
+      - name: Get Changes
+        id: changed-files
+        uses: tj-actions/changed-files@v35
+        with:
+          files: ./packages/icons/public/svg/**
+
+      - name: Exit if no changes
+        if: steps.changed-files.outputs.any_modified == 'false'
+        run: |
+          git checkout main
+          git branch --delete ${{ steps.current-branch.outputs.branch }}
+          exit 1
+
+      - name: Create Changeset
+        run: |
+          echo ---                                      > $CHANGESET
```

Not sure if we're missing a `>` here.

```suggestion
          echo ---                                      >> $CHANGESET
```